packages/types/package.json

```diff
@@ -21,7 +21,11 @@
     "test": "jest test",
     "type-check": "tsc --project ./tsconfig.build.json --noEmit"
   },
+  "peerDependencies": {
```

Comment:
Do the peer/dev dependencies result in circular dependency warning since caip imports types?